fission-cli/library/Fission/CLI/Parser/Config/IPFS.hs

```diff
@@ -49,5 +49,5 @@ timeoutParser =
     , long "ipfs-timeout"
     , metavar "SECONDS"
     ----------
-    , value 300
+    , value 1800
```

Comment:
Ran into this while testing the above with a large file. We have a more sophisticated timeout mechanism on the server now, so letting the CLI have a longer default timeout.